wizeline/remix-project-lab
# Feature/158/make people profiles accessible

# What does this PR do?

Make the profile of all users accessible to the current user.


# Where should the reviewer start?
Run the project locally and select a project, then click on the name of a user to go the selected user's profile info.


# How should this be manually tested?
- Go to home, select a user and check if its GitHub information is displayed
	- Username
	- Projects
- If the selected user haven't login recently the interface should display: " There is no information to display".


# What are the relevant tickets?

158

# Screenshots
![image](https://github.com/wizeline/remix-project-lab/assets/42613217/812a8599-6453-4cfe-9192-240a4993dfbf)

![image](https://github.com/wizeline/remix-project-lab/assets/42613217/54d36e83-1e9f-44f4-b8a7-4a4f34111e0f)

# Checklist
